### PR TITLE
Gitlab gem update + Make chandler run on `rake release`

### PR DESCRIPTION
**Pull request by orta**
Somehow the travis deploy script wasn't working, so now I've improved the `rake release` task to also run chandler if you have either one of those ENV vars in your shell.

### CODE REVIEW

**Comment by orta** [comment]
@k0nserv - does this look fine to you?

**Comment by orta** [comment]
And @connorshea - these api route changes feel right to you?

```diff
-        url = "https://gitlab.com/api/v3/projects/#{slug}/merge_request/#{merge_request_id}"
 +        url = "https://gitlab.com/api/v3/projects/#{slug}/merge_requests/#{merge_request_id}"
```

File: spec/support/gitlab_helper.rb [hunk 1]
```diff
@@ -27,19 +27,19 @@ def stub_request_source
 
       def stub_merge_request(fixture, slug, merge_request_id)
         raw_file = File.new("spec/fixtures/gitlab_api/#{fixture}.json")
-        url = "https://gitlab.com/api/v3/projects/#{slug}/merge_request/#{merge_request_id}"
```
Comment (k0nserv) [inline spec/support/gitlab_helper.rb]: I'm surprised that this worked at all, seems like I made a mistake here

Comment (orta) [inline spec/support/gitlab_helper.rb]: I think the gitlab gem was using that route, and as of 4.0.0 it now does the `s`

Comment (k0nserv) [inline spec/support/gitlab_helper.rb]: So in the API both worked?

Comment (orta) [inline spec/support/gitlab_helper.rb]: That's what I was hoping @connorshea would know :D

Comment (orta) [inline spec/support/gitlab_helper.rb]: I expect people are using both 3.x and 4.x in production ( as we migrated Danger last week) - I found the issue, looks correct https://github.com/NARKOZ/gitlab/issues/229

Comment (connorshea) [inline spec/support/gitlab_helper.rb]: yep, I think we should probably move to v4 soon but for now this is fine :)

**Pull request merged**